hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -231,6 +234,9 @@ public void close() throws IOException {
       RocksDBCheckpointDifferHolder
           .invalidateCacheEntry(rocksDBCheckpointDiffer.getMetadataDir());
     }
+    if (statistics != null) {
+      IOUtils.close(LOG, statistics);
+    }
     IOUtils.close(LOG, db);
```

Comment:
nit: `IOUtils.close` accepts multiple objects to be closed, and handles `null` values, too.

```suggestion
    IOUtils.close(LOG, db, statistics);
```